CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+<a name="Unreleased"></a>
+# [Unreleased](https://github.com/commercetools/ui-kit/compare/v1.0.0-beta.30...df570e9) (2018-09-06)
+
+
+### Bug Fixes
+
+* avoid machine-based paths in generated files ([004cd38](https://github.com/commercetools/ui-kit/commit/004cd38))
+* **select-input:** default selectInput to null ([635d48e](https://github.com/commercetools/ui-kit/commit/635d48e))
+* **select-input:** fix typo, and update code documentation ([5892497](https://github.com/commercetools/ui-kit/commit/5892497))
+* **select-input:** update code documentation ([f1a7e8d](https://github.com/commercetools/ui-kit/commit/f1a7e8d))
+
+
+### Features
+
+* **typography/text:** to accept isItalic for text detail and body ([df570e9](https://github.com/commercetools/ui-kit/commit/df570e9))
+
+
+
+<a name="1.0.0-beta.30"></a>
+# [1.0.0-beta.30](https://github.com/commercetools/ui-kit/compare/3ba7996...v1.0.0-beta.30) (2018-09-03)
```

Comment:
This release contains a huge amout of commits, probably due to the migration. In any case I felt like unnecessary to have them listed here. I added a couple of comments below instead 👇